visualization/app/codeCharta/ui/dialog/dialog.globalSettings.component.ts

```diff
@@ -166,19 +165,18 @@ export class DialogGlobalSettingsController
 
 	mapTrackingDataAvailable() {
 		const files = this.storeService.getState().files
-		return isStandalone() && isSingleState(files) && getVisibleFileStates(files)
+		return isSingleState(files) && getVisibleFileStates(files)
```

Comment:
Here is the same thing with the single state (mode). Maybe we should change it to multiple.

Reply:
This will be fixed by #2656